openassessment/workflow/models.py

```diff
@@ -574,10 +574,10 @@ def get_by_submission_uuid(cls, submission_uuid):
             return cls.objects.get(submission_uuid=submission_uuid)
         except cls.DoesNotExist:
             return None
-        except DatabaseError:
-            error_message = u"Error finding workflow for submission UUID {}.".format(submission_uuid)
-            logger.exception(error_message)
-            raise AssessmentWorkflowError(error_message)
+        except DatabaseError as exc:
```

Comment:
Note to reviewers: part of the previous change, trying to make error messages somewhat consistent.